test_runner/performance/test_physical_replication.py

```diff
@@ -164,7 +170,7 @@ def test_replication_start_stop(
                 master_connstr,
             ]
         )
-        replica_pgbench = [None for i in range(num_replicas)]
+        replica_pgbench: List[Optional[subprocess.Popen[Any]]] = [None for i in range(num_replicas)]
```

Comment:
Why even populate this list to begin with?

Reply:
So I can index into it